.golangci.yml

```diff
@@ -34,7 +34,8 @@ run:
   # won't be reported. Default value is empty list, but there is
   # no need to include all autogenerated files, we confidently recognize
   # autogenerated files. If it's not please let us know.
-  skip-files: []
+  skip-files:
+    - ".*_gen.go$"
```

Comment:
could we upstream an issue to the project?

Reply:
Already one filed at https://github.com/cheekybits/genny/issues/63